matter_demos.xml

```diff
@@ -1,723 +1,1153 @@
+		<property key="demos.blurb" value="Matter - Thermostat app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd2601b"/>
+		<property key="demos.imageFile" value="Examples/thermostat-app-thread/BRD2601B/chip-efr32-thermostat-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Thermostat Application over Thread for BRD2601B</description>
+	</demo>
+	<demo name="brd4187c.demo.light_switch_app_wifi_wf200" label="Matter - Light switch over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Light switch app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4187c"/>
+		<property key="demos.imageFile" value="Examples/light-switch-app-wifi/wf200/BRD4187C/chip-efr32-light-switch-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Light switch Application for BRD4187C to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4161a.demo.light_switch_app_wifi_wf200" label="Matter - Light switch over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Light switch app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4161a"/>
+		<property key="demos.imageFile" value="Examples/light-switch-app-wifi/wf200/BRD4161A/chip-efr32-light-switch-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Light switch Application for BRD4161A to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4186c.demo.light_switch_app_wifi_wf200" label="Matter - Light switch over Wi-Fi">
+		<property key="demos.blurb" value="Matter - Light switch app"/>
+		<property key="core.partCompatibility" value=".*"/>
+		<property key="core.boardCompatibility" value="brd4186c"/>
+		<property key="demos.imageFile" value="Examples/light-switch-app-wifi/wf200/BRD4186C/chip-efr32-light-switch-example.s37"/>
+		<property key="core.readmeFiles" value="https://github.com/SiliconLabs/matter#readme"/>
+		<property key="filters" value="Wireless\ Technology|Matter"/>
+		<property key="core.quality" value="PRODUCTION"/>
+		<description>This is a Matter Light switch Application for BRD4186C to be used with WF200 Wi-Fi expansion board</description>
+	</demo>
+	<demo name="brd4161a.demo.light_switch_app_wifi_rs9116" label="Matter - Light switch over Wi-Fi">
```

Comment:
Two binaries are missing:
1. light_switch_app_wifi_rs9116 for BRD4186C
2. light_switch_app_wifi_rs9116 for BRD4187C